content/v2.0/reference/api/influxdb-1x/dbrp.md

```diff
@@ -0,0 +1,98 @@
+InfluxDB 2.0 replaces both with [buckets](/v2.0/reference/glossary/#bucket).
+To support InfluxDB 1.x query and write patterns in InfluxDB 2.0, databases and retention
+policies are mapped to buckets using the **database and retention policy (DBRP) mapping service**.
+
+The DBRP mapping service uses the **database** and **retention policy** specified in
+[compatibility API](/v2.0/reference/api/influxdb-1x/) requests to route operations to a bucket.
+[Create DBRP mappings automatically](#dbrp-mapping-logic) with the `/write` compatibility endpoint or
+[create DBRP mappings manually](#manually-create-and-manage-dbrp-mappings) using the `/api/v2/dbrps` API endpoint.
+
+- [DBRP mapping behavior](#dbrp-mapping-behavior)
+- [Manually create and manage DBRP mappings](#manually-create-and-manage-dbrp-mappings)
+
+### Default retention policies
+A database can have multiple retention policies with one set as default.
+If no retention policy is specified in a query or write request, InfluxDB uses
+the default retention policy for the specified database.
+
+## DBRP mapping logic
+
+- [When writing data](#when-writing-data)
+- [When querying data](#when-querying-data)
+
+### When writing data
+When writing data to InfluxDB 2.0 using the [`/write` compatibility endpoint](/v2.0/reference/api/influxdb-1x/write/),
+the DBRP mapping service checks for a bucket mapped to the database and retention policy:
+
+- If a mapped bucket is found, data is written to the bucket.
+- If an unmapped bucket with a name matching:
+    - **database/retention policy** exists, a DBRP mapping is added to the bucket,
+      and data is written to the bucket.
+    - **database** exists (without a specified retention policy), the default
+      database retention policy is used, a DBRP mapping is added to the bucket,
+      and data is written to the bucket.
+- If no matching bucket is found, a new **database/retention-policy** bucket is
+  created with a DBRP mapping, and data is written to the bucket.
+  If no retention policy is specified, `autogen` is used.
+
+    {{% note %}}
+To automatically create new buckets, the authentication token used for the
+write request must be an **All Access token**.
+    {{% /note %}}
+
+### When querying data
+When querying data from InfluxDB 2.0 using the [`/query` compatibility endpoint](/v2.0/reference/api/influxdb-1x/query/),
+the DBRP mapping service checks for the specified database and retention policy
+(if no retention policy is specified, the database's default retention policy is used):
+
+- If a mapped bucket exists, data is queried from the mapped bucket.
+- If no mapped bucket exists, InfluxDB returns an error.
+
+## Manually create and manage DBRP mappings
+If you have existing buckets that don't follow the **database/retention-policy** naming convention,
+but you want to map databases and retention policies to them,
+use the using the [`/api/v2/dbrps` API endpoint](/v2.0/api/#tag/DBRPs) to
+manually create and manage DBRP mappings.
```

Comment:
I think this needs to be a BIG NOTE:

starting with --  naming convention, you **MUST** create a database and retention policy to them in order for you to query them via the compatibility API.  

Use the [`/api/v2/dbrps` API endpoint](/v2.0/api/#tag/DBRPs) to
manually create and manage DBRP mappings.